lambdatimes/src/components/Content/Content.js

```diff
@@ -18,39 +18,43 @@ export default class Content extends Component {
 
   componentDidMount() {
     // Once the component has mounted, get the data and reflect that data on the state.
+    this.setState({
+      tabs:tabData,
+      cards:cardData,
+      changeSelected:this.changeSelected,
+      filterCards:this.filterCards
+  })
   }
 
   changeSelected = tab => {
     // this function should take in the tab and update the state with the new tab.
-  };
+    return()=>{
+      this.setState({selected:tab});
+    }
```

Comment:
This shouldn't have the `return` statement. Should just be `this.setState({selected:tab});`